public/app/plugins/panel/candlestick/CandlestickPanel.tsx

```diff
@@ -313,6 +313,7 @@ export const CandlestickPanel: React.FC<CandlestickPanelProps> = ({
             )}
             {data.annotations && (
               <ExemplarsPlugin
+                visibleLabels={getVisibleLabels(config, [info.frame])}
```

Comment:
Is this necessary or should this property be optional? What does it do in this context?